.github/workflows/test.yml

```diff
@@ -1,4 +1,9 @@
-on: push
+on: 
+  push:
+    branches: [ "master" ]
```

Comment:
Any particular reason to only run these on `master`?

Reply:
> Any particular reason to only run these on `master`?

No, I just thought it would be nice to let actions only run on importance branches like master. But maybe better to keep the original behavior 14c5816